packages/graphin/src/components/Tooltip/useTooltip.tsx

```diff
@@ -54,15 +58,17 @@ const useTooltip = (props: Props) => {
     }
 
     /** 设置变量 */
-    setState(preState => {
-      return {
-        ...preState,
-        visible: true,
-        item: e.item,
-        x,
-        y,
-      };
-    });
+    timer.current = window.setTimeout(() => {
+      setState(preState => {
+        return {
+          ...preState,
+          visible: true,
+          item: e.item,
+          x,
+          y,
+        };
+      });
+    }, props?.delay?.show);
```

Comment:
Now that it is mandatory this should work
```suggestion
    }, props.delay.show);
```